openy_af4_vue_app/src/components/filters/Filters.vue

```diff
@@ -236,6 +240,12 @@ export default {
         !this.isEqual(this.selectedLocations, this.initialLocations) ||
         !this.isEqual(this.selectedActivities, this.initialActivities)
       )
+    },
+    headerClasses() {
+      return this.bsVersion === 4 ? 'd-none d-lg-flex' : 'hidden-md hidden-lg'
```

Comment:
It should have 'hidden-xs hidden-sm' for BS 3...

Reply:
@duozersk thanks for notice
fixed.
Please check it again